easybuild/easyconfigs/c/cuQuantum/cuQuantum-22.05.0.41-CUDAcore-11.4.2.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'Tarball'
+
+name = 'cuQuantum'
+version = '22.05.0.41'
+shortversion = '.'.join(version.split('.')[0:3])
+
+homepage = 'https://developer.nvidia.com/cuquantum-sdk'
+description = """NVIDIA cuQuantum SDK is a high-performance library for 
+quantum information science and beyond"""
+
+#cuStateVec requires CUDA 11.4 or above
+#python extension need scipy-stack
+#so gcccuda-2020.1.114
+toolchain = {'name': 'gcccuda', 'version': '2020.1.114'}
+toolchainopts = {'optarch': 'False'}
+
+# By downloading, you accept the cuQuantum Software License Agreement
+# (https://developer.nvidia.com/cuquantum-license-agreement)
+# accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/compute/cuquantum/redist/cuquantum/linux-x86_64/cuquantum-linux-x86_64-%s-archive.tar.xz' %version]
```

Comment:
You have twice the source archive. You can remove the one from `source_urls`.

Reply:
done